### PR TITLE
Remove Draft Programming Answer Feature from Assessment & Form

### PR DESCRIPTION
**Pull request by bivanalhar**

### CODE REVIEW

File: client/locales/en.json [hunk 1]
```diff
@@ -7898,4 +7892,4 @@
   "users.troubleSigningIn": {
     "defaultMessage": "Trouble signing in?"
   }
-}
```
Comment (cysjonathan) [inline client/locales/en.json:7901]: Trailing newline in en and zh jsons shouldn't be removed.

**Review by cysjonathan — CHANGES_REQUESTED** [review]
1. Check through the original PR: [Allow versioning of draft answer into past answers](https://github.com/Coursemology/coursemology2/pull/5802/files)
 #5802, there are some additional changes like `workflow_state`, `is_save_draft` which are introduced then, but not removed here.  
2. Also confirm what we want to do with "answers" which are in "attempting" state which are not evaluated (instead saved as draft), do these exist in our DB (reference staging DB), and do we want to delete them for consistency?
3. Since we're sure we don't want this feature, can also update the schema and drop the column in a migration file in this same PR.

File: client/locales/en.json (hunk 1 above)
Comment (bivanalhar) [reply client/locales/en.json:7901]: this is not the trailing newline. If you look into the file directly, you'll notice that the newline is still there

Comment (cysjonathan) [reply client/locales/en.json:7901]: 👍 oh it's not removed, it's added

File: client/app/bundles/course/assessment/submission/containers/PastAnswers.jsx [hunk 2]
```diff
@@ -100,10 +97,7 @@ class PastAnswers extends Component {
     return (
       <div key={answer.id}>
         <Typography variant="h6">
-          {answer.isDraftAnswer
-            ? intl.formatMessage(translations.savedAt)
```
Comment (cysjonathan) [inline client/app/bundles/course/assessment/submission/containers/PastAnswers.jsx:104]: missed the deletion of this translation

File: client/locales/en.json [hunk 3]
```diff
@@ -3083,9 +3077,6 @@
   "course.assessment.submission.downloadSubmissionsJobPending": {
     "defaultMessage": "Please wait as your request to download submission answers is being processed."
   },
-  "course.assessment.submission.draftAnswer": {
```
Comment (cysjonathan) [inline client/locales/en.json:3086]: missed equivalent deletion in zh.json

**Review by cysjonathan — CHANGES_REQUESTED** [review]
Minor nits, other than that, will need to rebase to resolve db/schema conflict for line 13: `ActiveRecord::Schema[7.2].define(version: xxx)`.

It's fine to use the latest version on master, since `db:migrate` will still be able to capture the earlier `20241129164745_remove_draft_programming_answer_column` migration.

File: client/app/bundles/course/assessment/submission/containers/PastAnswers.jsx (hunk 2 above)
Comment (bivanalhar) [reply client/app/bundles/course/assessment/submission/containers/PastAnswers.jsx:104]: done

**Review by cysjonathan — APPROVED** [review]
LGTM

**Pull request merged**